k8s/mongodb/mongo-ss.yaml

```diff
@@ -72,7 +72,7 @@ spec:
         resources:
           limits:
             cpu: 200m
-            memory: 768Mi
+            memory: 3.5G
```

Comment:
This Kubernetes docs page says that G and Gi are both okay:

https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-ram-container/#cpu-and-ram-units